js/quotes.js

```diff
@@ -1637,5 +1637,24 @@ const quotes = [
 	{
 		text: "Children are not colouring books. You don't get to fill them with your favourite colours.",
 		author: "Khaled Hosseini",
+	},
+  		text: "It's not what happens to you, but how you react to it that matters.",
```

Comment:
Hi! left curly (open) bracket missing here. 
```suggestion
{
  		text: "It's not what happens to you, but how you react to it that matters.",
```

Reply:
ops forgot it when I resolved the conflict! thanks!